lte/gateway/release/build-magma.sh

```diff
@@ -352,3 +353,6 @@ $(glob_files "${PY_TMP_BUILD}/usr/bin/*" /usr/local/bin/) \
 $(glob_files "${ANSIBLE_FILES}/config_stateless_*.sh" /usr/local/bin/)"
 
 eval "$BUILDCMD"
+
+cd "${PWD}"
+"${SCRIPT_DIR}"/build-ovs.sh "${OUTPUT_DIR}"
```

Comment:
Why are we doing this? I do not expect frequent OVS changes, so adding such step on every magma build would be wasteful.

Reply:
added change detection for OVS-related files and and environment variable override to force build